docs/reference/schema.md

```diff
@@ -75,6 +76,45 @@ export const protectedUsers = csTable("users", {
 });
 ```
 
+### Nested objects
+
+Protect.js supports nested objects in your schema, allowing you to encrypt **but not search on** nested properties. You can define nested objects up to 3 levels deep.
+This is useful for data stores that have less structured data, like NoSQL databases.
+
+You can define nested objects by using the `csValue` function to define a value in a nested object. The value naming convention of the `csValue` function is a dot-separated string of the nested object path, e.g. `profile.name` or `profile.address.street`.
```

Comment:
The column/attribute name in the DB table can't contain a `.` since we're using it a delimiter, right? Definitely an edge case, but thought it was worth mentioning.